src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -85,19 +86,19 @@ <h2 class="card-pf-title">
                           {{mission.name}}
                         </div>
                         <div class="list-group-item-text" *ngIf="mission.description">
```

Comment:
The mission description should extend almost to the end of the row. It currently stop 3/4 of the way.

Reply:
@mindreeper2420 have taken off the extra margin but still we are using `truncate` filter which is making it stop before , we can increase from 45 to more then it may go to next link for some resolutions.

Reply:
Looks better now - I'd leave the truncation alone then, if it is not handled via CSS. I resized my screen with the current changes and nothing overran or fell off of the cards, so it looks fine to me.